.travis.yml

```diff
@@ -7,7 +7,7 @@ services:
 
 env:
   - DIST=fedora28 PYTHON=python3 FLAKE='/usr/bin/flake8'
-  - DIST=fedoradev PYTHON=python3 FLAKE='/usr/bin/flake8'
+#  - DIST=fedoradev PYTHON=python3 FLAKE='/usr/bin/flake8'
```

Comment:
Why did this break?

Reply:
It is broken even on master, the docker image will not start installing stuff, some conflicts in the rawhide files. We can enable it again when it is fixed by Fedora team.

Reply:
Okay, then.